masonry/src/widget/tests/safety_rails.rs

```diff
@@ -16,9 +16,6 @@ fn make_parent_widget<W: Widget>(child: W) -> ModularWidget<WidgetPod<W>> {
             ctx.place_child(child, Point::ZERO);
             size
         })
-        .paint_fn(move |child, ctx, scene| {
-            child.paint(ctx, scene);
-        })
```

Comment:
Should this get at child paint (and maybe accessibility) in some other way?

Reply:
I don't think so - all that this paint implementation did was paint the child, which is now implicit/always the behaviour.

I think that this is fine.